build/package/Microsoft.DotNet.Cli.Installer.MSI.targets

```diff
@@ -24,6 +24,8 @@
         <CombinedFrameworkSdkHostInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(BundleExtension)</CombinedFrameworkSdkHostInstallerFile>
         <SdkInstallerNuspecFile>$(RepoRoot)/packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.$(Architecture).nuspec</SdkInstallerNuspecFile>
         <SdkInstallerNupkgFile>$(InstallerOutputDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).$(FullNugetVersion).nupkg</SdkInstallerNupkgFile>
+        <SdkInstallerNuspecV1File>$(RepoRoot)/packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.$(Architecture).v1.nuspec</SdkInstallerNuspecV1File>
+        <SdkInstallerNupkgV1File>$(InstallerOutputDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).v1.$(FullNugetVersion).nupkg</SdkInstallerNupkgV1File>
```

Comment:
Why do we need both? Is it for VS 15.0.x servicing?

Reply:
Correct. We have to continue servicing 15.0.